extensions/shared/src/main/java/app/revanced/extension/youtube/patches/components/FeedComponentsFilter.java

```diff
@@ -166,12 +166,20 @@ public FeedComponentsFilter() {
 
         channelProfileGroupList.addAll(
                 new ByteArrayFilterGroup(
-                        Settings.HIDE_VISIT_COMMUNITY_BUTTON,
+                        Settings.HIDE_COMMUNITY_BUTTON,
                         "community_button"
                 ),
                 new ByteArrayFilterGroup(
-                        Settings.HIDE_VISIT_STORE_BUTTON,
-                        "header_store_button"
+                        Settings.HIDE_STORE_BUTTON,
+                        "store_button"
```

Comment:
It's not that important, but I'll use the previous keyword.

This is the accessibility id, and YouTube 20.22+ requires this keyword for this feature to work properly.